full-service/src/json_rpc/v2/api/request.rs

```diff
@@ -230,6 +230,9 @@ pub enum JsonCommandRequest {
         first_block_index: Option<String>,
         next_subaddress_index: Option<String>,
     },
+    reimport_account {
```

Comment:
I think we should call this "resync_account" or something, as it doesn't have anything to do with importing an account